src/CarbonAware.WebApi/src/Models/EmissionsDataForLocationsParametersDTO.cs

```diff
@@ -0,0 +1,18 @@
+using CarbonAware.Aggregators.CarbonAware;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace CarbonAware.WebApi.Models;
+
+public class EmissionsDataForLocationsParametersDTO : CarbonAwareParametersBaseDTO
+{
+    /// <summary>String array of named locations</summary>
+    /// <example>eastus</example>
+    [BindProperty(Name = "location"), BindRequired] override public string[]? MultipleLocations { get; set; }
```

Comment:
Notes: Use the same patterns as we have in follow-on PRs

```suggestion
    [FromQuery(Name = "location"), SwaggerParameter(Required = true)] public override string[]? MultipleLocations { get; set; }
```

Reply:
Updated